doc/installation/mac_install.rst

```diff
@@ -1,95 +1,120 @@
+Building NEST on macOS requires some developer tools. There are several sources from
+which you can install them, e.g., Conda, Homebrew, or MacPorts. The most important
+recommendation for an easy and stable build is *not to mix tools from different sources*.
+This includes your Python installation: Taking Python from Conda and all else from Homebrew
+may work, but can also lead to various complications.
 
-The clang/clang++ compiler that ships with OS X/macOS does not support OpenMP threads and creates code that fails some tests. You therefore need to use **GCC** to compile NEST under OS X/macOS.
+This guide shows how to build NEST with a development environment created with Conda. The main
+advantage of Conda is that you can fully insulate the entire environment in a Conda environment.
+If you want to base your setup on Homebrew or MacPorts, you can still use the ``extras/conda-env-nest-simulator.yml``
+file as a guide to necessary packages.
 
-Installation instructions here have been tested under macOS 10.14 *Mojave* with `Anaconda Python 3 <https://www.continuum.io/anaconda-overview>`_ and all other dependencies installed via `Homebrew <http://brew.sh>`_. They should also work with earlier versions of macOS.
+Preparations
+------------
 
-#.  Install Xcode from the AppStore.
+1. Install the Xcode command line tools by executing the following line in the terminal and 
+   following the instructions in the windows that will pop up:
 
-#.  Install the Xcode command line tools by executing the following line in the terminal and following the instructions in the windows that will pop up:
+   .. code-block:: sh
 
- .. code-block:: sh
+      xcode-select --install
 
-        xcode-select --install
+#. Create a conda environment with necessary tools (see also :doc:`conda_tips`)
 
-#.  Install dependencies via Homebrew:
+   .. code:: sh
 
- .. code-block:: sh
+      conda env create -f extras/conda-environment-nest-simulator.yml
 
-       brew install gcc cmake gsl open-mpi libtool
+   .. note::
 
-#.  Create a directory for building and installing NEST (you should always build NEST outside the source code directory; installing NEST in a "place of its own" makes it easy to remove NEST later).
+      To build NEST natively on a Mac with Apple's M1 chip, you need to use Miniforge as 
+      described in :doc:`conda_tips`.
 
-#.  Extract the NEST tarball as a subdirectory in that directory or clone NEST from GitHub into a subdirectory:
+#. Activate the environment with
 
- .. code-block:: sh
+   .. code:: sh
+   
+      conda acvitate nest-simulator
+      
+   This assumes that you have created the environment with its default name ``nest-simulator``.
 
-        mkdir NEST       # directory for all NEST stuff
-        cd NEST
-        tar zxf nest-simulator-x.y.z.tar.gz
-        mkdir bld
-        cd bld
+#. If you want to build NEST with MPI, you must codesign the ``orterun`` and ``orted`` binaries
```

Comment:
```suggestion
#. If you want to build NEST with MPI, you must digitally sign the ``orterun`` and ``orted`` binaries
```
Rationale: I misread *codesign* as *co-design* here because the context at this point did not give me enough of a clue. Then I was confused.

Alternatively, put `codesign` in double backticks to make clear that this is a command.